lib/docker-file.ts

```diff
@@ -26,10 +26,45 @@ async function readDockerfileAndAnalyse(
   return analyseDockerfile(contents);
 }
 
+function resolveVariables(
+  text: string,
+  instruction: Instruction,
+  dockerfile: Dockerfile,
+): string {
+  const variables = instruction.getVariables();
+  if (variables) {
```

Comment:
No need for this condition. Looking at the libraries code getVariables will always return a list. We should try to reduce cyclomatic complexity whenever possible.

Reply:
Done, thanks